src/components/sidebar/tree.js

```diff
@@ -73,7 +73,15 @@ const Tree = ({edges}) => {
   const [treeData] = useState(() => {
     return calculateTreeData(edges);
   });
-  const [collapsed, setCollapsed] = useState({});
+  const defaultCollapsed = {};
+  treeData.items.forEach(item => {
+    if (config.sidebar.collapsedNav.includes(item.url)) {
```

Comment:
To maintain backwards compatibility and to handle users forgetting to put `collapsedNav` in their config.js file, can we make a check to see if `collapsedNav` exists before applying methods like `includes`? More like `if(config.sidebar.collapsedNav && config.sidebar.collapsedNav.includes(item.url))`

Reply:
Nice catch! I updated this to make sure this setting exists before collapsing a sidebar item by default.